### DISCUSSION TITLE
Errors connecting to  ODBC connection

### DISCUSSION THREAD

**Discussion by Arpitha387** [opendistro-for-elasticsearch/sql#1128] [Q&A]
I have downloaded ODBC sql driver 1.13 and Elastic Search version is 7.11.0
Still am getting error like failed to connection DB

![image](https://user-images.githubusercontent.com/87110280/124866346-e6529080-dfd9-11eb-8c0a-8cd996b3bff3.png)

Please help to fix this issue

**Comment by anirudha** [comment]
What do you mean by Elasticsearch? Do you have the opendistro sql plugin installed?

Please add more Details about your setup

**Comment by hhoosscchhii** [comment]
I have a similar concerns and am curious to find out more about the emitted error code: 

![image](https://user-images.githubusercontent.com/89050933/129643659-f6a7f640-6c18-4fea-aef2-16d91ec1ca51.png)
